examples/voting/src/Wallet.js

```diff
@@ -0,0 +1,144 @@
+/* eslint-disable no-alert */
+/* eslint-disable no-console */
+/* eslint-disable @typescript-eslint/return-await */
+/* eslint-disable consistent-return */
+/* eslint-disable react/prop-types */
+/* eslint-disable react/jsx-filename-extension */
+/* eslint-disable import/no-unresolved */
+import React from 'react';
+import { detectConcordiumProvider } from '@concordium/browser-wallet-api-helpers';
+import { Alert, Button } from 'react-bootstrap';
+import { AccountTransactionType, GtuAmount, ModuleReference } from '@concordium/web-sdk';
+import moment from 'moment';
+import { RAW_SCHEMA_BASE64, TESTNET_GENESIS_BLOCK_HASH } from './config';
+
+export async function init(setConnectedAccount) {
+    const client = await detectConcordiumProvider();
+    // Listen for relevant events from the wallet.
+    client.on('accountChanged', (account) => {
+        console.debug('browserwallet event: accountChange', { account });
+        setConnectedAccount(account);
+    });
+    client.on('accountDisconnected', () => {
+        console.debug('browserwallet event: accountDisconnected');
+        client.getMostRecentlySelectedAccount().then(setConnectedAccount);
+    });
+    client.on('chainChanged', (chain) => {
+        console.debug('browserwallet event: chainChanged', { chain });
+    });
+    client.getMostRecentlySelectedAccount().then(setConnectedAccount);
+
+    return client;
+}
+
+export async function connect(client, setConnectedAccount) {
+    const account = await client.connect();
+    return setConnectedAccount(account);
+}
+
+// Check if the user is connected to the testnet chain by checking if the testnet genesisBlock exists.
+// The smart contract voting module is deployed on the testnet chain.
+async function checkConnectedToTestnet(client) {
+    return await client
```

Comment:
```suggestion
/* eslint-disable consistent-return */
/* eslint-disable react/prop-types */
/* eslint-disable react/jsx-filename-extension */
/* eslint-disable import/no-unresolved */
import React from 'react';
import { detectConcordiumProvider } from '@concordium/browser-wallet-api-helpers';
import { Alert, Button } from 'react-bootstrap';
import { AccountTransactionType, GtuAmount, ModuleReference } from '@concordium/web-sdk';
import moment from 'moment';
import { RAW_SCHEMA_BASE64, TESTNET_GENESIS_BLOCK_HASH } from './config';

export async function init(setConnectedAccount) {
    const client = await detectConcordiumProvider();
    // Listen for relevant events from the wallet.
    client.on('accountChanged', (account) => {
        console.debug('browserwallet event: accountChange', { account });
        setConnectedAccount(account);
    });
    client.on('accountDisconnected', () => {
        console.debug('browserwallet event: accountDisconnected');
        client.getMostRecentlySelectedAccount().then(setConnectedAccount);
    });
    client.on('chainChanged', (chain) => {
        console.debug('browserwallet event: chainChanged', { chain });
    });
    client.getMostRecentlySelectedAccount().then(setConnectedAccount);

    return client;
}

export async function connect(client, setConnectedAccount) {
    const account = await client.connect();
    return setConnectedAccount(account);
}

// Check if the user is connected to the testnet chain by checking if the testnet genesisBlock exists.
// The smart contract voting module is deployed on the testnet chain.
function checkConnectedToTestnet(client) {
    return client
```

Removes the rule /* eslint-disable @typescript-eslint/return-await */
by removing the redundant await in the `return await client`